examples/DataWizard/Polygon Regions DataWizard.ipynb

```diff
@@ -0,0 +1,758 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Working with Data wizard - using some of the polygon queries. \n",
+    "\n",
+    "Data Wizard is the new python API way to interact with instance of apertureDB.\n",
+    "\n",
+    "It tries to avoid the need to know the details of the native query mechanism\n",
+    "\n",
+    "It is still under development, so not all the capabilities of native query would be available.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Simplest query."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using native Query"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from aperturedb import Connector\n",
+    "from aperturedb.Images import Images\n",
+    "\n",
+    "db = Connector.Connector(user=\"admin\", password=\"admin\", port=55557)\n",
```

Comment:
I think we should always use dbinfo.create_connector() in the examples, so that people follow that convention. 
we would need to add a link to dbinfo.py here, like in the other example folders.

Reply:
this comment is applicable to all cells in the notebook.